apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.ts

```diff
@@ -73,6 +74,21 @@ router.get(
       return;
     }
 
+    if (req.query.qid && typeof req.query.qid === 'string') {
```

Comment:
We could instead support a route like `.../questions/qid/$qid`. Armando had suggested that this could be more RESTful.

Reply:
That does seem preferable to me. I can't see an advantage in the query string approach?

Reply:
Done in b9e1b0b.